src/stack_aggregator.py

```diff
@@ -591,8 +594,10 @@ def get_dependency_data(epv_set):
     epv_data = tr_epv_list['result']['data']
     epv_list, unknown_deps_list = find_unknown_deps(epv_data, epv_list,
                                                     tr_list, unknown_deps_list, True)
+    logger.info('Unknown Dependencies {ud}'.format(ud=unknown_deps_list))
 
     result = add_transitive_details(epv_list, epv_set)
+    logger.info('Result with Transitive details: {ud}'.format(ud=result))
```

Comment:
rather than this, i would like you to add a log for the final json that we construct. print that. it will be more helpful

Reply:
Ack!